test cases/unit/25 shared_mod linking/installed_files.txt

```diff
@@ -0,0 +1 @@
+usr/bin/prog?exe
```

Comment:
This file is not needed for unit tests. It is only used for project tests to verify that their installation step works. This test does not do any installing to this file is not needed.